integrations/zoom.js

```diff
@@ -1,47 +1,85 @@
 const fetch = require('node-fetch');
 
 module.exports = async function(robot, kredits) {
+  const Contributor = kredits.Contributor;
+  const Contribution = kredits.Contribution;
 
-  function createContributionFor (participant, meeting) {
-    // TODO
+  const kreditsContributionAmount = 500;
+  const kreditsContributionKind = 'community';
+
+  const zoomAccessToken = process.env.KREDITS_ZOOM_JWT;
+
+  const walletTransactionCount = await kredits.provider.getTransactionCount(kredits.signer.address);
+  let nonce = walletTransactionCount;
+
+  function createContributionFor(participant, meeting) {
+    const displayName = participant.name;
+
+    return getContributorByZoomDisplayName(displayName)
+      .then(contributor => {
+        let contribution = {
+          contributorId: contributor.id,
+          contributorIpfsHash: contributor.ipfsHash,
+          amount: kreditsContributionAmount,
+          kind: kreditsContributionKind,
+          description: `Team meeting: ${meeting.topic}`,
```

Comment:
This is a bit of an issue, because we sometimes use the Weekly Call room for other meetings. Not sure what the best solution is. Maybe just something generic like "Team/Community Call".